app/models/event.rb

```diff
@@ -1,10 +1,57 @@
 # frozen_string_literal: true
 
+class EventCallbacks
```

Comment:
Two options to maybe organise this:

1. Extract the EventCallbacks class into it's own file in models. Probably makes organising unit tests clearer as well.
2. Move the after_create method into Event, e.g.

```
class Event < Application Record
...
after_create ->(event) { check_support_messages(event) }
...
private
# renamed from after_create
def check_support_messages(new_event)
...
end
```